samples/python/app.py

```diff
@@ -1,6 +1,8 @@
 """
 This example integrates the IdPyOIDC library with the MIRACL Trust platform.
 """
+
+import argparse
```

Comment:
Do we need that? The omission of flags was intentional to make the sample shorter and easier to understand.

Reply:
It's easier to be run with flags and similar to the rest of the samples. I don't think it complicates it.